lib/src/repo.rs

```diff
@@ -965,3 +978,14 @@ impl MutableRepo {
         );
     }
 }
+
+/// Error from attempts to check out the root commit for editing
+#[derive(Debug, Copy, Clone, Error)]
+#[error("Opened root commit")]
```

Comment:
I saw that a test case fails. I think the "Cannot rewrite the root commit" is much clearer than this message. That message seems to make sense here. "Cannot edit the root commit" would also be fine (if you think "rewrite" is inappropriate at this abstraction level). I don't really have a preference.

Reply:
Reworded as advised.